include/bluetooth/gatt.h

```diff
@@ -917,6 +917,43 @@ static inline int bt_gatt_notify(struct bt_conn *conn,
 	return bt_gatt_notify_cb(conn, &params);
 }
 
+/** @brief Notify attribute value change by UUID.
+ *
+ *  Send notification of attribute value change, if connection is NULL notify
+ *  all peer that have notification enabled via CCC otherwise do a direct
+ *  notification only on the given connection.
+ *
+ *  The attribute object on the parameters is the starting point for the search
```

Comment:
i think we should remove "on the parameters", the attribute is not part of a params struct.

Reply:
Agreed